src/snek/game.rs

```diff
@@ -37,23 +53,63 @@ impl Game {
       .into_iter()
       .map(|food| (food, GameCoordinate::random(dimensions)))
       .collect();
+
+    let mut snake = Snake::new();
+    snake.grow(SnakeDirection::North);
+    snake.grow(SnakeDirection::North);
+    snake.grow(SnakeDirection::North);
+    snake.grow(SnakeDirection::East);
+    snake.grow(SnakeDirection::East);
+    snake.grow(SnakeDirection::East);
+
     Game {
       dimensions,
-      snek: Snake::new(),
+      snek: snake,
```

Comment:
lol at some point we should have the constructor generate a snake with one segment starting in the center but could be future pr

Reply:
yeah i figure future PR is alright for this - it like helps the demo make sense if nothing else